tests/accounts.rs

```diff
@@ -39,6 +39,57 @@ mod index {
 
         Ok(())
     }
+
+    #[test(harness = set_up)]
+    async fn as_admin_token(app: DivviupApi) -> TestResult {
+        let account = fixtures::admin_account(&app).await;
+        let other_account = fixtures::account(&app).await;
+        let (_, header) = fixtures::api_token(&app, &account).await;
+        let mut conn = get("/api/accounts")
+            .with_api_headers()
+            .with_request_header(KnownHeaderName::Authorization, header)
+            .run_async(&app)
+            .await;
+        let accounts: Vec<Account> = conn.response_json().await;
+
+        assert_eq!(accounts.len(), 2);
+
+        assert_eq!(accounts, vec![account, other_account]);
```

Comment:
Nit: redundant asserts? `Vec`s should be equal if their lengths and contents match.

Reply:
addressed in rebase